src/main/java/org/spongepowered/common/mixin/plugin/entityactivation/EntityActivationRange.java

```diff
@@ -443,7 +444,7 @@ public static void addEntityToConfig(World world, SpongeEntityType type, byte ac
         entityType = EntityActivationRange.activationTypeMappings.get(activationType);
         final String entityModId = type.getModId().toLowerCase();
         final String entityId = type.getName().toLowerCase();
-        EntityActivationRangeCategory activationCategory = config.getConfig().getEntityActivationRange();
+        EntityActivationRangeCategory activationCategory = globalConfig.getConfig().getEntityActivationRange();
         EntityActivationModCategory entityMod = activationCategory.getModList().get(entityModId);
         Integer defaultActivationRange = activationCategory.getDefaultRanges().get(entityType);
```

Comment:
The value should still be read from the `config` instance, to allow for per-world overrides.

Reply:
Nevermind, doesn't apply here.